lib/season_stats.rb

```diff
@@ -3,21 +3,43 @@
 require "pry"
 
 class SeasonStats < StatDaddy
-  def winningest_coach
-    @games_teams_fixture_path.each do |data|
-      # Name of the Coach with the best win percentage for the season
-    end
+
+  # kind of pseudo coded out this stuff
+  # @games.season.sort(season)
```

Comment:
I think you can just find_all where @games.season.include?("20122013") for example